Marsiple/Common/Network/HTTPResponseCodes.swift

```diff
@@ -9,39 +9,39 @@
 import Foundation
 
 enum HTTPResponseCode: Int {
```

Comment:
We should use the [official naming](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) for each HTTP status code. 

Most of them are good, but you use `successWithoutResponseBody` instead of `No content`. Intentional?

Reply:
I guess i found incorrect naming. Both `200` and `204` had naming `success` so i wanted to differentiate them.

Reply:
Understood.